Release Notes/2017/2017-08-01-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,60 @@
+
+    Learn more about Cloud Application Manager [here](https://www.ctl.io/cloud-application-manager/).
+
+### Managed Services
+
+    For CenturyLink Managed Services customers, we have enhanced our graphing and metrics reporting capabilities along with our AWS CloudWatch Monitoring capabilities in our recently launched monitoring portal.
+
+    In additional to customers being able to graph for CPU, Memory, and Disk metrics for Network has been included for comprehensive visibility into their infrastructure across all supported Operating Systems. Additionally, customers are now able to overlay any individual collected metric attribute to facilitate a comprehensive assessment of the behavior of their applications.
+
+    We also have enhanced our capabilities to natively monitor and alert on AWS services such as RDS, EC2 and ELB for example via CloudWatch, eliminating the requirement for an agent on the target device. If you would like to take advantage of our AWS CloudWatch monitoring functionality and Managed Services capabilities, please reach out to us via the ‘support center’ option within Cloud Application Manager.
+
+    Learn more about CenturyLink Managed Services [here](https://www.ctl.io/product-overview/#managed-services).
+
+### Ecosystem-Partner Integrations (3)
+
+The [CenturyLink Marketplace](https://www.ctl.io/marketplace/) added three new commercially available offerings with the following Partners:
+
+* [*OSNEXUS QuantaStor SDS EE with Gold Support*](https://www.ctl.io/marketplace/partner/ZV5T/product/QuantaStor%20SDS%20with%20Gold%20Support%20(License%20Only) - QuantaStor is a unified Software Defined Storage (SDS) platform designed to scale out to make storage management easy while reducing overall enterprise storage costs. This is a License Only purchase. The customer is responsible for deploying QuantaStor SDS separately on the infrastructure of their choosing.
+
+* [*Apex Datacom Call Me*](https://www.ctl.io/marketplace/partner/APEX/product/Call%20Me%20Express/) - Apex Call Me is a Click-to-Call solution that enables web marketers to convert more web visitors into customers. Apex Call Me works with any business phone system with a US or Canadian phone number.
+
+* [*Alert Logic Threat Manager with Log Review*](https://www.ctl.io/marketplace/partner/PST/product/Alert%20Logic%20Log%20Manager%20with%20Log%20Review/) - Alert Logic Threat Manager with Log Review is a cloud-based managed network intrusion detection and vulnerability assessment solution delivered as-a-service.
+
+### Announcements (2)
+
+### Object Storage
+
+**Expanded availability for CenturyLink Cloud Object Storage in Germany**
+
+CenturyLink is now able to sell service to customers who have a geographic preference for Object Storage service delivered from the Frankfurt CLC (DE1) datacenter in Germany. The existing product features and pricing apply to this new location.
```

Comment:
cloud storage is a geo redundant service and has a primary & secondary location.  thus we should never refer to any cloud storage as residing in a specific datacenter but simply say our new germany regional endpoint is available with the same redundant, highly durable storage